nextflow.config

```diff
@@ -31,7 +31,7 @@ params {
     report_contributors     = null
     report_description      = null
     report_scree            = true
-    report_round_digits     = 4
+    report_round_digits     = -1
```

Comment:
You want the rounding off by default now?

Reply:
Yeah that's an excellent question...I discussed this again with a colleague. I'm going to reenable the rounding by default for now. Let's see if this leads to further confusions with very small p values, then I might turn it off by default.